### PR TITLE
Sils/lints pack2

### PR DESCRIPTION
**Pull request by sils1297**
CC @AbdealiJK for review

### CODE REVIEW

File: bears/typescript/TSLintBear.py [hunk 1]
```diff
@@ -1,17 +1,33 @@
...
+@linter(executable='tslint')
+class TSLintBear:
+    """
+    Checks the code with ``tslint`` on each file separately.
+    """
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         tslint_config: path = "",
+                         rules_dir: path = ""):
```
Comment (AbdealiJK) [inline bears/typescript/TSLintBear.py]: We normally don't put spaces around the `=` here. Not sure if that's part of styleguide

Comment (sils1297) [inline bears/typescript/TSLintBear.py]: nope, changing

**Comment by sils1297** [comment]
unack 77da88e

**Comment by AbdealiJK** [comment]
ack
d8362183e6a0b8b2f207e43889e15763b467f5a5
e2fd3f727a3972e153f5cfc07b8b7d0da4767278
f02e3238552c82e5c063292d9e5172da07c998a2
33bc3b71a0f1e0c568e82d79c58287eacf23e29a
ca2547c3456cc3d263e6e71fe9036b139f631633
8ce40abfe95a7e87c240eff6417a142f467a0d0d
d355f6cd8e0bf6cca9d7eedd0c101ee5cc5d5b77
6f709585c9d0e4ebb4d54793b4bd8a80f090bfe2
2f26512cb7df49eed05e7e9d1c1004f2e113ab46
f0ad878e5889ab130c81d8352b781b2aca7012bf
0369ad5108dc0015962fad910140723156f7f29f
34bf7b0b90587d023772b7a77a25d3458af0a26e

**Comment by AbdealiJK** [comment]
ack d8362183e6a0b8b2f207e43889e15763b467f5a5 e2fd3f727a3972e153f5cfc07b8b7d0da4767278 f02e3238552c82e5c063292d9e5172da07c998a2 33bc3b71a0f1e0c568e82d79c58287eacf23e29a ca2547c3456cc3d263e6e71fe9036b139f631633 8ce40abfe95a7e87c240eff6417a142f467a0d0d d355f6cd8e0bf6cca9d7eedd0c101ee5cc5d5b77 6f709585c9d0e4ebb4d54793b4bd8a80f090bfe2 2f26512cb7df49eed05e7e9d1c1004f2e113ab46 f0ad878e5889ab130c81d8352b781b2aca7012bf 0369ad5108dc0015962fad910140723156f7f29f 34bf7b0b90587d023772b7a77a25d3458af0a26e

**Comment by sils1297** [comment]
ack
d836218
e2fd3f7
f02e323
33bc3b7
ca2547c
8ce40ab
d355f6c
6f70958
2f26512
f0ad878
0369ad5
34bf7b0

GitMate? You there? :'( ?

**Comment by sils1297** [comment]
reack
7a168c6 SQLintBear: Improve documentation
8e5cc44 SQLintBear: Refactor to Linter v2 API
83caa2a YAMLLintBear: Add LANGUAGES attribute
eb297e9 GNUIndentBear: Elaborate on language support
b5f292c CSharpLintBear: Improve documentation
07e2b6a CSharpLintBear: Add LANGUAGES attribute
8885784 CPPLintBear: Add LANGUAGES attribute
e94b989 CPPCleanBear: Add LANGUAGES attribute
f4783cb TSLintBear: Improve documentation
f02e323 VintBear: Add LANGUAGES attribute
e2fd3f7 VintBear: Update documentation
d836218 VintBear: Refactor to Linter v2 API

**Comment by sils1297** [comment]
reack eadc38d as per abdealis request

**Comment by sils1297** [comment]
ack
9fb5620 Add .codecov.yml file
f0ec7d9 YAMLLintBear: Add __init__.py file

**Comment by sils1297** [comment]
(coverage is full)

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**